ReactiveLocation/Classes/ReactiveLocation.swift

```diff
@@ -1,76 +1,76 @@
 import CoreLocation
-import ReactiveCocoa
+import ReactiveSwift
 import Result
 
-public enum LocationError: ErrorType {
-    case LocationError(CLError)
+public enum LocationError: Error {
+    case locationError(CLError.Code)
 }
 
 public enum RegionState {
```

Comment:
This wasnt changed in this PR, but I think this enum represents a `RegionEvent`, not a `RegionState`. Maybe now is a good time to make this breaking change?